### PR TITLE
Initial GitHub actions testing

### PR DESCRIPTION
**Pull request by RafalKorepta**
Introduce some testing for the helm chart.

- Cleanup whitespace
- Fix maintainer - should be a github user, not an arbitrary string
- Introduce testing
- Add test states to readme

It's based on #17 and #18

### CODE REVIEW

**Comment by RafalKorepta** [comment]
The pipeline passed here https://github.com/RafalKorepta/helm-charts/runs/1736284671?check_suite_focus=true

**Review by rkruze — CHANGES_REQUESTED** [review]
Should we move cluster.yaml and ct.yaml to under the .github/workflow directory?

**Comment by BenPope** [comment]
> Should we move cluster.yaml and ct.yaml to under the .github/workflow directory?

`redpanda/ci` seems good.

File: cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+kind: Cluster
```
Comment (BenPope) [inline cluster.yaml:1]: Is this used in github actions?

I guess it's based on what I did for local cluster testing?

File: cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+apiVersion: kind.x-k8s.io/v1alpha4
+containerdConfigPatches:
+  - |-
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:5000"]
```
Comment (senior7515) [inline cluster.yaml:5]: wow this string is unreal. never seen a `"foo".bar.baz."postfix"` in my life. crazy. is this a bug?

Comment (BenPope) [reply cluster.yaml:5]: https://kind.sigs.k8s.io/docs/user/local-registry/

File: cluster.yaml (hunk 1 above)
Comment (RafalKorepta) [reply cluster.yaml:1]: Yes, I will remove this in the next push.

File: cluster.yaml (hunk 2 above)
Comment (RafalKorepta) [reply cluster.yaml:5]: This file will be removed in the next push.

**Comment by RafalKorepta** [comment]
I address the comments. `ct.yaml` is moved to `redpanda/ci`. I reorganize the commits. The unused file where removed.

File: .github/workflows/lint-test.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Lint and Test Charts
+
+on: [push, pull_request]
```
Comment (0xdiba) [inline .github/workflows/lint-test.yml:12]: Let's limit `push` runs to `main` branch only.

Comment (RafalKorepta) [reply .github/workflows/lint-test.yml]: Ok. Done

File: .github/workflows/lint-test.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
```
Comment (rkruze) [inline .github/workflows/lint-test.yml:3]: This should be Apache 2 license

Comment (RafalKorepta) [reply .github/workflows/lint-test.yml]: No problem. Changed.

**Review by BenPope — APPROVED** [review]

**Review by rkruze — APPROVED** [review]

**Pull request merged**